DEVELOPMENT.md

```diff
@@ -0,0 +1,83 @@
+## Setup
+
+### Prerequisites
+
+  * [libgit2](https://libgit2.org/)
+    * macOS: `brew instal libgit2`
+
+## Unit Tests
+
+#### Run
+
+```
+make unit
+```
+
+## Local Registry
+
+#### Prerequisites
+
+  * [docker](https://docs.docker.com/get-docker/)
+  * [ngrok](https://ngrok.com/) - or any public routing solution
+
+1. In one terminal, start a docker registry
+    ```shell
+    docker run -p 5000:5000 --rm -e REGISTRY_STORAGE_DELETE_ENABLED=true registry:2
+    ```
+2. In another terminal, route port 5000:
```

Comment:
Is ngrok needed? For local k8s clusters like k3s etc it seems to work with localhost as well. Should we add this as an optional step?

Reply:
Does k3s also handle SSL for you? I use ngrok (and recommend it) because it handles all of that for you. If you know what you are doing then you would probably be able to skip these steps altogether.